lib/post_message_definition.yml

```diff
@@ -122,6 +122,10 @@ post_messages:
             type:
               code: string
               guid: string
+      backToSearch:
+        payload:
+          <<: *user_session_properties
```

Comment:
For clarity of the needs, which scenarios need the user_session_properties?  The use case I'm testing for only needs the context, so it would be nice to know the additional needs.

Reply:
I just added them to be consistent with the rest of connect postmessages. They simply add `user_guid` and  `session_guid` in .